packages/playwright/bundles/babel/src/babelBundleImpl.ts

```diff
@@ -26,7 +26,19 @@ export { parse } from '@babel/parser';
 import traverseFunction from '@babel/traverse';
 export const traverse = traverseFunction;
 
-function babelTransformOptions(isTypeScript: boolean, isModule: boolean, pluginsPrologue: [string, any?][], pluginsEpilogue: [string, any?][]): TransformOptions {
+
+interface BabelTransformOptionsInput {
+  isTypeScript: boolean
+  isModule: boolean
+  pluginsPrologue: [string, any?][]
+  pluginsEpilogue: [string, any?][]
+  jsx: string | undefined
```

Comment:
Love the input type (we would use type not interface for it). I'd hide jsx details into a nested struct

```js
jsx: { type, factory, fragmentFactory, importSource }
```

I'd also type the jsx type as enum if at all possible.